pom.xml

```diff
@@ -95,7 +100,7 @@
                     <plugin>
                         <groupId>org.sonatype.plugins</groupId>
                         <artifactId>nexus-staging-maven-plugin</artifactId>
-                        <version>1.6.13</version>
+                        <version>${nexus-staging-maven-plugin.version}</version>
```

Comment:
Just curious, what is the advantage of specifying the version via a property?

Reply:
I went back and forth between specifying them in property or just moving them out of property completely or moving it all to `dependency-management` section. I was aiming for some consistency while I went through and upgraded some of the plugins and dependencies as we had mixture.